www/md_base/src/app/builds/builder/builder.route.coffee

```diff
@@ -0,0 +1,15 @@
+class State extends Config
+    constructor: ($stateProvider) ->
+
+        # Name of the state
+        name = 'builds.builder'
+
+        # Register new state
+        $stateProvider.state
+            controller: "builderController"
+            controllerAs: 'builder'
+            templateUrl: "views/#{name}.html"
+            name: name
+            url: "/builder/:builderid"
+            data:
+                title: 'Builds / Builder'
```

Comment:
The title could be just `Builder` and you can use the parent state to get `Builds / Builder`.